module/Decision/src/Decision/Controller/FileBrowser/FileNode.php

```diff
@@ -64,4 +72,61 @@ public function getName()
     {
         return $this->name;
     }
+
+    /**
+     * Extension of file or dir
+     * @return string valid name
+     */
+    public function getExtension()
+    {
+        return $this->extension;
+    }
+
+    /**
+     * Get corresponding FontAwesome file icon
+     * @return string valid FontAwesome 5 icon
+     */
+    public function getFileIcon()
+    {
+        $types = array(
```

Comment:
Did we really need this? Also, please use the short array notation (`[..]`) as we're trying to standardize on that.

Reply:
I saw a stub in `files.phtml` where I thought it was tried having a dynamic file icon. Since I thought people would appreciate these icons, I included it. 
Moreover, at this point in time I don't see any drawbacks for not implementing it.